### PR TITLE
Metrics CoinRanking

### PR DESCRIPTION
**Pull request by LarryIVC**
### **In this PR, I did the following:**

- [ ]  Create a branch.
- [ ]  Add the Components.
- [ ]  Connect React and Redux.
- [ ]  Handle events in a React app.
- [ ]  Use store, actions, and reducers in React.
- [ ]  Use API to get data and metrics.
- [ ]  Add the logic.
- [ ]  Use styles in a React app.
- [ ]  Add the tests.
- [ ]  Check the linters.

### CODE REVIEW

File: src/components/Details.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  if (error) {
+    return (
+      <p>
+        Error:
+        {error}
+      </p>
+    );
+  }
+
+  // console.log(coinDetail);
```
Comment (afanimartin) [inline src/components/Details.js]: - Please remove this and any comments. Comments make our code ugly

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @LarryIVC, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Video recorded and link added to readme file
- App link added to readme file. App works as described
- Tests added and run with no errors
- Redux well configured

## Required Changes ♻️
_Check the comments under the review._
- Remove the back button from the Home page. Clicking on it does nothing and the original design doesn't have it

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Details.js (hunk 1 above)
Comment (LarryIVC) [reply src/components/Details.js]: Ok, is it mandatory?
The Microverse time is over.
Thanks, @afanimartin.

Comment (LarryIVC) [reply src/components/Details.js]: ## Hi @afanimartin, about the changes required.

> Remove the back button from the Home page. Clicking on it does nothing and the original design doesn't have it

Solved:
Home
image

Details
image

**Review by julie-ify — APPROVED** [review]
Hi @LarryIVC,

## Highlights!
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working routes ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**